lib/appmap/open.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module AppMap
+  OpenStruct = Struct.new(:appmap)
+
+  class Open < OpenStruct
+    attr_reader :port
+
+    def perform
+      server = run_server
+      open_browser
+      server.kill
+    end
+
+    def page
+      require 'rack/utils'
+      <<~PAGE
+      <!DOCTYPE html>
+      <html>
+      <head>
+        <title>&hellip;</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <script type="text/javascript">
+        function dosubmit() { document.forms[0].submit(); }
+        </script>
+      </head>
+      <body onload="dosubmit();">
+        <form action="https://app.land/scenario_uploads" method="POST" accept-charset="utf-8">
```

Comment:
Is there any utility in allowing the user to choose a server here (e.g. in the environment)?